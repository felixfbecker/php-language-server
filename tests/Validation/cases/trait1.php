<?php

class TestClass {
    use TestTrait;
}
