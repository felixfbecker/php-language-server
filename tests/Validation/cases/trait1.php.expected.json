{
    "references": {
        "TestTrait": [
            "./trait1.php"
        ]
    },
    "definitions": {
        "TestClass": {
            "fqn": "TestClass",
            "extends": [
                "TestTrait"
            ],
            "isMember": false,
            "roamed": false,
            "isStatic": false,
            "canBeInstantiated": true,
            "symbolInformation": {
                "name": "TestClass",
                "kind": 5,
                "location": {
                    "uri": "./trait1.php"
                },
                "containerName": ""
            },
            "type": null,
            "declarationLine": "class TestClass {",
            "documentation": null,
            "signatureInformation": null
        }
    }
}